docs/participant_endpoint_enums.md

```diff
@@ -0,0 +1,37 @@
+# Participant Endpoint Enums
+
+This is the list of endpoints that a PISP or DFSP must register with the swith to get callbacks from the `thirdparty-api-adapter` and `transaction-request-service`.
+
+These endpoints should be added to the enums of `central-services-shared`, and the seeds of `central-ledger`
+
+## PISP:
+
+- `THIRDPARTY_CALLBACL_URL_CONSENT_REQUEST_PUT`
+- `THIRDPARTY_CALLBACK_URL_CONSENT_REQUEST_PUT_ERROR`
+- `THIRDPARTY_CALLBACK_URL_CONSENT_POST`
+- `THIRDPARTY_CALLBACK_URL_CONSENT_REQUEST_PUT`
+- `THIRDPARTY_CALLBACK_URL_TRANSACTION_REQUEST_PUT`
```

Comment:
sorry can I ask where we use this callback? because iirc for the transfer flow we only need the following callbacks for the success case:
- `PUT /parties/{Type}/{ID}`
- `POST /authorizations`
- `PUT /transfers/{ID}`

Reply:
Yeah you're right. Since we're still deciding on the design, it It may be for a`PUT /thirdpartyRequests/transactions/{ID}` instead of a `PUT /transfers/{ID}` I wanted it here just in case.